src/ShipEngine.php

```diff
@@ -178,11 +169,11 @@ public function removeCarrier(CarrierCode $carrierCode, string $carrierId): bool
     public function listCarrierServices(string $carrierId)
     {
         $endpoint = sprintf("carriers/%s/services", $carrierId);
```

Comment:
Noticed you changed some of these to single quotes and left some as double quotes. Any reason for that?

Reply:
Old code and I guess we never cleaned up. I organized